iceoryx_examples/iceoptions/README.md

```diff
@@ -81,6 +84,10 @@ If we want to enforce the contract that the publisher needs to support a `histor
 `requirePublisherHistorySupport` to `true`. In this case the subscriber will only connect if the publisher history support is at least as large as is request.
 By default this is set to `false` and best-effort behavior is used.
 
+!!! warning
+    In case of n:m communication, the history feature will **not** provide the overall last n samples based on delivery point in time!
+    For more information about this limitation see the [QoS article](https://iceoryx.io/v2.0.0/concepts/qos-policies/).
```

Comment:
I have to strongly disagree here, sorry!

This is what we have done in the past to avoid relative links. There are several links to the iceoryx.io already in the examples. The reason is the following:

The `Readme.md`s of all examples are currently the only `.md` which are shared between GitHub and [iceoryx.io](https://iceoryx.io).

For iceoryx.io the links have to look like:
```
[QoS article](qos-policies.md) // mkdocs-autolinks-plugin will resolve the link
#or
[QoS article](../../doc/website/concepts/qos-policies.md)
#or
[QoS article](https://iceoryx.io/v2.0.0/concepts/qos-policies/)
```

For reading the document via GitHub's Web GUI:
```
[QoS article](https://iceoryx.io/v2.0.0/concepts/qos-policies/)
#or
[QoS article](../../doc/website/concepts/qos-policies.md) // User would land on a GitHub page with broken links
#or
[QoS article](https://iceoryx.io/v2.0.0/concepts/qos-policies/)
```

The decision for `v1.0` was to avoid relative links and use absolute links. IMHO we should stick to it.

cc @FerdinandSpitzschnueffler @dkroenke

Reply:
I would keep this as it was for 1.0 and then we need to create some tooling around this to fix it.

Reply:
@mossmaurice Then please use the link `[QoS article](../../doc/website/concepts/qos-policies.md)`

As you mentioned it works on the website and the markdown document and for 3.0 we create some tooling to clean this up.

Reply:
@elBoberido The reason is in my comment above:

`[QoS article](../../doc/website/concepts/qos-policies.md) // User would land on a GitHub page with broken links`

Additionally, using relative links turned out to be a debacle with `v1.0.0`. Hence, the conclusion was to remove them.

Reply:
Okay, then let's keep it as with v1 also for v2. It's an inconvenience to be redirected from github to the homepage but the links are working